### DISCUSSION TITLE
enableCursorHighlight doesn't always work.

### DISCUSSION THREAD

**Discussion by DezyNation** [heroui-inc/heroui#927] [Feedback]
Hey everyone,
I'm using NextUI for my next project. I have created a simple Navbar and I'm using  `enableCursorHighlight` and I think it doesn't always work. The hover effect is becomes non-functional after just a few lines of code update.
Am I the only one who is facing this issue?

**Comment by Siumauricio** [comment]
Yes, this is an issue, but in production doesn't happen.
